xarray/tests/test_cftimeindex_resample.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import
+
+import pytest
+
+import datetime
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+pytest.importorskip('cftime')
+pytest.importorskip('pandas', minversion='0.23.99')
```

Comment:
In addition to merging in the master branch, could you please also update the `minversion` to 0.24 here now that pandas 0.24 is officially out?  Thanks!